packages/desktop-ui-lib/src/lib/time-tracker/time-tracker.component.ts

```
@@ -449,10 +449,13 @@
 		try {
 			const { lastTimer, isStarted } = timer;
 
-			const isRemote =
-				this._timeTrackerStatus.remoteTimer &&
-				this.xor(!isStarted, this._timeTrackerStatus.remoteTimer.running) &&
-				this._startMode === TimerStartMode.REMOTE;
+			const remoteTimer = this._timeTrackerStatus.remoteTimer;
+			const isRemoteTimerRunning = remoteTimer && remoteTimer.running;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)